src/main/java/ua/ukr/net/Dao/JdbcEmployeeDao.java

```diff
@@ -1,16 +1,10 @@
 package ua.ukr.net.Dao;
```

Comment:
Rename please package Dao to dao. Package name should start with lower case letter